### PR TITLE
Workflow with no outputs to delete should transition to terminal state [49_hotfix edition] [WA-149]

### PR DESCRIPTION
**Pull request by salonishah11**
JIRA issue: https://broadworkbench.atlassian.net/browse/WA-149

### CODE REVIEW

File: centaur/src/main/resources/standardTestCases/no_output_delete.test [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# When there are no files to delete, Cromwell should successfully transition to terminal state
+
+name: no_output_delete
+testFormat: workflowsuccess
+backends: [Papiv2]
+
+files {
+  workflow: delete_intermediates/no_output_delete.wdl
+  options: delete_intermediates/delete_intermediates.options
```
Comment (cjllanwarne) [inline centaur/src/main/resources/standardTestCases/no_output_delete.test]: I don't see this file in this PR?

Comment (salonishah11) [reply centaur/src/main/resources/standardTestCases/no_output_delete.test]: It is already [exists](https://github.com/broadinstitute/cromwell/blob/develop/centaur/src/main/resources/standardTestCases/delete_intermediates/delete_intermediates.options) as part of previous tests.

**Comment by cjllanwarne** [comment]
Is this also tested for a workflow with no tasks at all?

**Comment by salonishah11** [comment]
@cjllanwarne I have added the `workflow-with-no-task` test in the [develop](https://github.com/broadinstitute/cromwell/pull/5458) version of this PR.

**Pull request merged**